weldx/__init__.py

```diff
@@ -1,4 +1,106 @@
-"""WelDX data format and analysis package."""
+"""
+WelDX - data and quality standards for welding research data
+============================================================
+
+The weldx package provides several tools to setup an experiment, systematically process
+measurement data, and archive it in a standardized fashion.
+
+Here we quickly list the most important functions and classes.
+
+
+**File handling**
+
+Read and write weldx files.
+
+.. autosummary::
+    :toctree: _autosummary
+    :caption: File handling
+    :template: class-template.rst
+
+    WeldxFile
+
+**Welding**
+
+These classes and functions are used to define welding processes.
+
+.. autosummary::
+    :toctree: _autosummary
+    :caption: Welding
+
+    GmawProcess
+    get_groove
+
+
+**Geometry**
+
+These classes are used to define workpiece geometries.
+
+.. autosummary::
+    :toctree: _autosummary
+    :caption: Geometry
+    :template: class-template.rst
+
+    ArcSegment
+    Geometry
+    LineSegment
+    LinearHorizontalTraceSegment
+    Profile
+    Shape
+    Trace
+    SpatialData
+
+
+**Data handling and transformation**
+
+.. autosummary::
+    :toctree: _autosummary
+    :caption: Data handling and transformation
+    :template: class-template.rst
```

Comment:
```suggestion
.. autosummary::
    :toctree: _autosummary
    :caption: Data handling and transformation
    :template: class-template.rst
    :noindex:
```

can `:noindex` help here with removing the duplicate links warnings?

Reply:
Thanks for the hint, I will try it out

Reply:
both directive options do not exist.

Reply:
I found it here https://github.com/sphinx-doc/sphinx/issues/7052 https://github.com/sphinx-doc/sphinx/pull/7906 but not sure if we can use it

Reply:
The `:noindex:` directive is listed here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#basic-markup

not sure if we can set this with `autosummary`, I guess you tried already @marscher ?
Maybe we have to create another template for this. It would be nice not to have index entries for the user API entries